README.md

```diff
@@ -159,3 +156,52 @@ The package `toilet` cannot be installed yet. But TBomb does still work.
 For Queries: [Telegram Group](https://t.me/TBombChat)  
 [Check Out My YouTube Channel](https://www.youtube.com/c/SpeedXTech)
 
+## FAQ
+
+⁃ Internet Connection poor:
+
+```Check your connection and try again. Maybe ping any adress to be sure its not your line.```
```

Comment:
Address is misspelled. 

Also, rephrase the sentence to `Check your internet connection and try pinging any remote address to confirm`.